docs/RFCs/private-network.md

```diff
@@ -0,0 +1,66 @@
+- Feature Name: private-network
+- Start Date: 2022-02-01
+- RFC PR:
+- Functionland Issue:
+
+## Background
+we are using IPFS as our file system. but IPFS is build to use for public data, and it does not support ACL, 
+so we need to find a way to keep user safe until our security layer become mature.
+
+## Problem Statement
+We need to protect user and his/her data from harms and risk of public network.  
+
+## Motivation
+Isolating User from public network can help us relax the problem and build useful product and testing our security layer without putting user in harms way. 
+
+## Proposal
+using IPFS build-in private network. base on [spec](https://github.com/libp2p/specs/blob/master/pnet/Private-Networks-PSK-V1.md)
+Its uses a pre-shared key (PSK) to create a private network with encrypted communication.
+
+For box setup user provide an environment variable SECRET which is password he should remember.
+the secret then convert to a hash of 256 bit by algorithm like sha256 and generate the swarm.key for ipfs and libp2p node's.
+
+For Fula client when user call createClient he should also provide the password he used for setting the box's.
```

Comment:
Consider setting the scene before talking specifically about Fula client. e.g. a brief sentence reminding the reader what Fula client is and how the client operates in the private network, and whether its operation is any different.


```suggestion
For Fula client when user calls `createClient` they should also provide the password used for setting up the boxes.
```

* Use backtick for code and ideally link it up with API docs.
* Ditto re neutral pronouns.
* I recommend also setting the scene before this paragraph about the second secret. 

Question: Why two secrets at all? Why not use that secret for key generation as well?  The answer to these questions should be included in the doc.